app/frontend/src/features/communities/events/EventPage.tsx

```diff
@@ -162,7 +167,9 @@ export default function EventPage() {
           eventKey(eventId),
           updatedEvent
         );
-        queryClient.invalidateQueries(eventKey(eventId));
+        queryClient.invalidateQueries(eventKey(eventId), {
+          refetchActive: false,
+        });
```

Comment:
Can't remember from memory whether our other API mutations also return back the updated object we can use to update the cache (with "optimistic" update, in quotes since it's done in `onSuccess` rather than `onMutate`), but if it does, could be good to follow this to reduce API refetch calls from invalidating the query, since the returned result from the mutation should be "fresh enough" without needing to check API again straight away.